### DISCUSSION TITLE
Will there be no more releases?

### DISCUSSION THREAD

**Discussion by Raffa064** [termux/termux-app#3868] [General]
Termux 0.118.0 has not been updated for 2 years.

Will there be any more updates? Is this version of f-droid updated with this repository?

I hope you continue to update because this project is really cool, and very useful in some cases.

**Comment by TomJo2000** [comment]
Please see:
- #3848
and
- #3213
for more information, but to summarize.

Termux 0.118 is a close to feature complete terminal emulator, so an update of the core app has not been necessary for quite a while.
Aside from that the packages are maintained separately.
